modules/gdscript/gdscript.cpp

```diff
@@ -905,6 +745,15 @@ Error GDScript::reload(bool p_keep_state) {
 			return err;
 		}
 	}
+
+#ifdef TOOLS_ENABLED
+	// Done after compilation because it needs the GDScript object's inner class GDScript objects,
+	// which are made by calling make_scripts() within compiler.compile() above.
+	if (parser.get_tree()->identifier) { // Only generate documentation for named classes.
```

Comment:
Why, though? Currently we generate it for unnamed classes as well, IIRC. Does it not work well? Docs for unnamed classes should be helpful for inspector tooltips at the very least.

Reply:
I was really convinced this wasn't the case, but I believe you're right from a quick look through the old code. I'll dig into it a lil deeper and make sure this works as expected!

Reply:
I can't find the code that I thought made it so we didn't generate docs for unnammed classes, which probably just means it wasn't there to begin with ;)

I removed this check :)